stdlib/3/unittest/case.pyi

```diff
@@ -215,6 +215,10 @@ class TestCase:
                            exception: Union[Type[_E], Tuple[Type[_E], ...]],
                            expected_regex: Union[str, bytes, Pattern[str], Pattern[bytes]],
                            msg: Any = ...) -> _AssertRaisesContext[_E]: ...
+    def assertDictContainsSubset(self,
+                                 expected: Dict[Any, Any],
```

Comment:
The argument type should be Mapping, not Dict. It may be possible to do something more precise than Any for the key and value types, but `Any` is fine with me.

Reply:
Should I sync this with the copy in `stdlib/2/unittest.pyi`? (That's where I copied the definition from.)

Reply:
Yes, please.